From ab5af66e551639a7c4bcb4467b746518639ab514 Mon Sep 17 00:00:00 2001
From: Hans de Goede <johannes.goede@oss.qualcomm.com>
Date: Sat, 13 Dec 2025 12:25:10 +0100
Subject: [PATCH] Makefile: Add -fPIC to CFLAGS

Building a PIE executable requires the object files to be build with -fPIC,
add the missing -fPIC to CFLAGS.

Signed-off-by: Hans de Goede <johannes.goede@oss.qualcomm.com>
---
 Make.defaults | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Make.defaults b/Make.defaults
index 581d7e6..8aea65c 100644
--- a/Make.defaults
+++ b/Make.defaults
@@ -9,6 +9,7 @@ CFLAGS += -DCOLOR_NORMAL=0x0f
 CFLAGS += -fno-lto
 CFLAGS += '-DEFI_MACHINE_TYPE_NAME="x64"'
 CFLAGS += '-DGIT_VERSION="1.0-ubuntu0"'
+CFLAGS += -fPIC
 CFLAGS += -O2
 
 LDFLAGS += -nostdlib
-- 
2.52.0

